tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -10,41 +10,43 @@
 import boto3
 from typing import Dict, Any, List, Optional
 from datetime import datetime, timezone
-import shared_recovery
-import requests_db
-import database
+from shared_recovery import (
+    OrcaStatus,
+    RequestMethod,
+    post_status_for_file_to_queue,
+    post_status_for_job_to_queue,
+)
+from requests_db import get_dbconnect_info
+from database import single_query, result_to_json
 from requests_db import DatabaseError
+from cumulus_logger import CumulusLogger
 
-PREFIX = 'dev'
-DB_HOST = 'dev-drdb-host'
-DATABASE_NAME = 'postgres'
-DATABASE_PORT = '5432'
-DATABASE_USER = 'rhassan'
-OS_ENVIRON_DB_QUEUE_URL_KEY = 'DB_QUEUE_URL'
-OS_ENVIRON_RECOVERY_QUEUE_URL_KEY = 'DB_QUEUE_URL_RECOVERY'
+# instantiate CumulusLogger
+LOGGER = CumulusLogger()
 
+PREFIX = os.environ["PREFIX"]
```

Comment:
Generally, I would not make these variables global. In reality you do not need to set the following variables in your code since the `db_connect_info = get_dbconnect_info()` does this for you.
```
PREFIX
DATABASE_NAME
DATABASE_PORT
DATABASE_USER
```

The remaining two variables DB_QUEUE_URL and RECOVERY_QUEUE_URL I would get values for in the handler and perform proper checking to make sure they are set. This includes logging and adding a raise Exception event if the values are not present or of 0 length.